### PR TITLE
Address ssc issue 870

### PR DESCRIPTION
**Pull request by sjanzou**

### CODE REVIEW

**Comment by sjanzou** [comment]
Tests added and updated for pvwattsv8, summary document attached along with reference.
[ssc_870.zip](https://github.com/NREL/ssc/files/9606935/ssc_870.zip)

File: test/ssc_test/cmod_json_comparison_test.h [hunk 1]
```diff
@@ -58,11 +58,15 @@ class JSONComparisonTest : public ::testing::Test {
 
 public:
     void Test(const std::string& compute_module, const std::string& file_inputs, const std::string &file_outputs, const std::vector<std::string> &compare_number_variables, const std::vector<std::string> &compare_array_variables) {
+        char solar_resource_path[256];
```
Comment (brtietz) [inline test/ssc_test/cmod_json_comparison_test.h:61]: Looks like the error tolerance on lines 92 and 101 is too tight for the linux vs mac differences. Is a blanked increase to 0.01 ok, or do we need to do this on a percentage basis?

**Review by brtietz — CHANGES_REQUESTED** [review]
Looks like this fixes the problem- having the inverter efficiency as a time series output is very helpful. Just need to update some tolerances to the mac tests pass as well.

File: test/ssc_test/cmod_json_comparison_test.h (hunk 1 above)
Comment (sjanzou) [reply test/ssc_test/cmod_json_comparison_test.h:61]: tolerance updated for macOS tests. All tests now pass
image

**Review by brtietz — APPROVED** [review]

**Comment by cpaulgilman** [comment]
https://github.com/NREL/ssc/issues/870

**Pull request merged**